tests/test_ugrid.cpp

```diff
@@ -704,6 +704,29 @@ TEST( MeshUgridTest, magnitude_direction )
   }
 }
 
+TEST( MeshUgridTest, VoidMesh )
+{
+  std::string path = test_file( "/ugrid/meshCreation/void_mesh.nc" );
+
+  std::string uri = "Ugrid:\"" + path + "\":" + "mesh2d";
+  std::string uriToMeshNames = "Ugrid:\"" + path + "\"";
+
+  EXPECT_EQ( MDAL_MeshNames( uriToMeshNames.c_str() ), "Ugrid:\"" + path + "\":mesh2d" );
+  MDAL_MeshH m = MDAL_LoadMesh( uri.c_str() );
+
+  ASSERT_NE( m, nullptr );
+  MDAL_Status s = MDAL_LastStatus();
+  EXPECT_EQ( MDAL_Status::None, s );
+
+  int v_count = MDAL_M_vertexCount( m );
+  EXPECT_EQ( v_count, 0 );
```

Comment:
I have a question, if I have void mesh and want to add a single face (I would like to see in the auto-tests to see that we have f_count=1 afterwards), I need to RECREATE the whole file from scratch or I can just "update" it??

I think in general we want to have a new capability on driver if they are able to update the mesh too? I imagin for netcdf-based formats it should be possible.?. and it would be kind of nice to not need to recreate it from scratch?

Reply:
For any update, you need to rewrite the whole file. It seems difficult for now to allow updating only parts of the file for several reasons.
But indeed, it will be a good new capability and it will be possible with the netcdf based format. But, I think it will need quite big changes on the way the mesh is handle for this format For, now it is a memory mesh, so for updating only some parts we need to sync the memory mesh with the mesh in the file.